src/gsy_e/models/strategy/energy_parameters/energy_params_eb.py

```diff
@@ -119,6 +324,21 @@ def event_traded_energy(
             product_type: One of the available market types.
         """
         assert self._profile_generator is not None
+
+        if product_type == AvailableMarketTypes.INTRADAY:
```

Comment:
Minor, and just see this no: Why not naming it `market_type`?

Reply:
@hannesdiedrich I think this was a decision that changed a few times (it was "market type" initially, then "exchange", then we settled on "product type"). I think the important thing would be to always use the same naming though, so to replace the occurrences of `market_type` with `product_type`.

Reply:
:-( ok, Thanks! I really wonder why `market type` was not suitable any more. Anyway. Will name it `product_type` from now on then.